content/influxdb/cloud/get-started.md

```diff
@@ -1,299 +1,77 @@
 
-_See [Differences between InfluxDB Cloud and InfluxDB OSS](#differences-between-influxdb-cloud-and-influxdb-oss)._
+To get started working in Cloud, we recommend using demo data and notebooks. Complete the following steps:
+1. [Add demo data](add-demo-data)
+2. ...
 
-## Start for free
+## Add demo data
 
-Start using {{< cloud-name >}} at no cost with the [Free Plan](/influxdb/cloud/account-management/pricing-plans/#free-plan).
-Use it as much and as long as you like within the plan's rate-limits.
-Limits are designed to let you monitor 5-10 sensors, stacks or servers comfortably.
+This example uses InfluxDB Cloud's Website Monitoring demo data bucket. To add the demo data bucket:
 
-## Sign up
+1.  In the navigation menu on the left, click **Data (Load Data)** > **Buckets**.
 
-1. Choose one of the following:
+    {{< nav-icon "data" >}}
 
-    #### Subscribe through InfluxData
+2.  Click **{{< icon "plus" >}} Add Demo Data**, and then select the **Website Monitoring** bucket.
+3.  The Demo Data bucket appears in your list of buckets.
 
-    To subscribe to an InfluxDB Cloud **Free Plan** through InfluxData,
-    go to [InfluxDB Cloud](https://cloud2.influxdata.com/).
+## Select metrics
 
-    - To use social sign-on, click **Google** or **Microsoft**. Note that social sign-on does not support email aliases.
-    - Sign up with email by entering your name, email address, and password, then click **Create Account**.
+Now that we've added the demo data bucket, we're ready to create our notebook. The first notebook cell selects the bucket and metrics we'll work with.
 
-    If you originally signed up with email but want to enable social sign-on, you can do so by logging in through your cloud provider as long as you use the same email address.  
+1. In the navigation menu on the left, click **Notebooks**.
 
-    #### **Subscribe through a cloud provider**  
-    To subscribe to an InfluxDB Cloud **Usage-Based** plan and pay through your
-    **Amazon Web Services (AWS)**, **Microsoft Azure**, or **Google Cloud Platform (GCP)** account:
+    {{< nav-icon "notebooks" >}}
+2. Click **+Create Notebook**.
+3. Enter a name for your notebook in the **Name this notebook** field.
+4. By default, a **Metric Selector** and **Visualization** cell appear (see [Cell types](#cell-types) below for details.)
+5. In the **Metric Selector** cell, click the **Select a bucket** dropdown and choose **Website Monitoring Bucket**.
 
-    - **AWS**  
-      Sign in to AWS, navigate to the [InfluxDB Cloud product on AWS Marketplace](https://aws.amazon.com/marketplace/pp/B08234JZPS/?href=_ptnr_web_docs_gettingstarted),
-      and follow the prompts to subscribe. After you click **Set Up Your Account**,
-      enter your credentials, and then click **Start Now**.
-      All usage charges will be paid through the subscribed AWS account.
+{{< img-hd src="/img/influxdb/gs_select_bucket.png" alt="Select bucket" />}}
```

Comment:
- I'm guessing you need to indent this to properly nest this in the `li` and not break the `ol` numbering.
- Update the image file name to use the [image naming conventions](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#image-naming-conventions):

    ```
    /img/influxdb/cloud-gs-select-bucket.png
    ```